libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -187,16 +187,15 @@ method unsubscribe*(p: PubSub,
                     topics: seq[TopicPair]) {.base, async.} =
   ## unsubscribe from a list of ``topic`` strings
   for t in topics:
+    # metrics
+    libp2p_pubsub_topics.dec()
```

Comment:
is this really the right place? also, to avoid miscounts, there's `set` instead of inc/dec